server/server.go

```diff
@@ -56,14 +57,16 @@ func NewRodeServer(
 	opa opa.Client,
 	esClient *elasticsearch.Client,
 	filterer filtering.Filterer,
+	elasticsearchRefresh config.RefreshOption,
```

Comment:
sure. good idea. I realized we were using all of the other config stuff outside of server.go. But definitely for the future in case we add more settings. fixing it now!